packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -1092,7 +1100,8 @@ describe('scope hoisting', function() {
       assert(!/.-./.test(contents));
     });
 
-    it('removes unused transpiled classes using terser when minified', async function() {
+    it.skip('removes unused transpiled classes using terser when minified', async function() {
+      // TODO: SWC needs to add PURE annotations
```

Comment:
(Can be reenabled once https://github.com/swc-project/swc/pull/1639 is merged)

Reply:
I think this was pure annotations for transpiled classes which I think are still missing